src/source/mainwindow.cpp

```diff
@@ -247,3 +247,32 @@ void MainWindow::on_actionLogout_triggered()
     emit adminFeaturesToggled(false);
 }
 
+/*!
+ * \brief MainWindow::on_actionAdd_new_stadium_triggered
+ * @brief Adds stdaium with its corresponding attritbutes
+ * @brief Adds edges with weights between Lags Vegas and it surrounding cities.
+ * @brief Adds souviers to Lag Vegas stdium
+ *
+ */
+void MainWindow::on_actionAdd_new_stadium_triggered()
+{
+    db->AddStadium("Las Vegas Stadium","Las Vegas Gamblers","123 Las Vegas Blv,NV 89101","(702) 962-4000","2016-04-11","50,000","Grass",0,"American","Modern");
+    int id =  db->GetStadiumID("Las Vegas Stadium");
+    qDebug()<<"ID; "<<id;
+    db->AddDistance(id,7,325);
```

Comment:
the IDs for the stadiums shouldn't be hardcoded, use db->GetStadiumID with the appropriate stadium names to acquire those IDs. just to make sure stuff doesn't get mucked up.